### ISSUE TITLE
Some weights of the PyTorch model were not used when initializing the TF 2.0 model TFBertForTokenClassification:

### ISSUE THREAD

**Issue by GhaziSultan** [GhaziSultan/Front-end#1]
I am a student and I am new to NLP.  I want to do to Named Entity on scientific articles. For that purpose, I am using TFBertForTokenClassification with "allenai/scibert_scivocab_cased". 

tokenizer = AutoTokenizer.from_pretrained('allenai/scibert_scivocab_cased')
model=TFBertForTokenClassification.from_pretrained("allenai/scibert_scivocab_uncased",from_pt=True,config=config)

But it gives following warning:
**Some weights of the PyTorch model were not used when initializing the TF 2.0 model TFBertForTokenClassification: ['classifier.weight', 'classifier.bias'] - This IS expected if you are initializing TFBertForTokenClassification from a TF 2.0 model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a TFBertForPretraining model). - This IS NOT expected if you are initializing TFBertForTokenClassification from a TF 2.0 model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a TFBertForSequenceClassification model)**

And when I try to fit the model, it shows the following warning:
history = model.fit(train_ds, epochs=1, validation_data=devel_ds)

**WARNING:tensorflow:Gradients do not exist for variables ['tf_bert_for_token_classification/bert/pooler/dense/kernel:0', 'tf_bert_for_token_classification/bert/pooler/dense/bias:0'] when minimizing the loss.**

Can you please tell me why it is showing these warnings and how to solve this issue?
